docs/project/code_review.md

```diff
@@ -124,15 +124,8 @@ mechanical best practices to most effectively navigate them.
     -   Make any in-file comments needed, but add them to a pending review
         rather than sending them directly.
     -   Finish the review and add any top-level review comments there.
--   If you are an owner who will be providing approval for the change, then make
-    sure to mark a review as requesting changes when you want the author to
-    begin addressing your comment. Only use the "comment" review state if you
-    are still in the process of reviewing and don't expect the author to begin
-    working on further changes.
-    -   If you are not an owner asked to approve, use the difference between a
-        comment and requesting a change to help the author know whether to
-        circle back with you before landing the pull request if the relevant
-        owner(s) approve it.
+-   Don't mark a review as requesting changes - this feature doesn't work well
+    for Carbon's review workflows. Only use the "comment" review state.
```

Comment:
FYI, the issue is that once you have requested changes it won't turn that indicator off until the requester selects "approve." It does not support passing the baton back and forth or other state evolution (waiting for another to give feedback, sending feedback part way through reviewing, and so on).

Reply:
Worth adding some wording to that effect, or no?

Something like:

>  Don't mark a review as requesting changes - this state can only be cleared by approval from the original reviewer, making other workflows impossible. Only use the "comment" review state.

Reply:
Where you say "original reviewer", I think request changes means that _you_ need to approve, not another reviewer. (or, the code author needs to be comfortable dismissing your review)

So maybe something like:

> Only mark a review as requesting changes if you are talking with the reviewer and still need to approve before merging. Otherwise, it will block merge even if someone else approves.

Reply:
Sorry, I'm trying to aim for a subtly different phrasing, but maybe that was a little too loose. Maybe:

> We don't mark reviews as requesting changes. Sometimes reviewers get busy and somebody else approves; the "request changes" feature still blocks merges until the original requester approvers, which can slow reviews down.